src/Microsoft.TestPlatform.CoreUtilities/Helpers/DotnetHostHelper.cs

```diff
@@ -58,7 +57,7 @@ public string GetDotnetHostFullPath()
                 }
             }
 
-            string errorMessage = String.Format(Resources.NoDotnetExeFound, dotnetExeName);
+            string errorMessage = String.Empty; //String.Format(Resources.NoDotnetExeFound, dotnetExeName);
```

Comment:
I would need to add resource file for this project to accommodate for this resource.